src/main/java/seedu/address/model/person/Name.java

```diff
@@ -10,13 +10,13 @@
 public class Name {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Names should only contain alphanumeric characters and spaces, and it should not be blank";
+            "Names should only contain alphabetic characters and spaces, and it should not be blank";
```

Comment:
Would this be better?
```suggestion
            "Names should only contain alphabets and spaces, and it should not be blank";
```

Reply:
Good suggestion! I'll make the changes